config/config.json

```diff
@@ -448,17 +464,25 @@
                         "config": {}
                     }
                 }
+            },
+            "telemetry" : {
+                "enabled":true,
+                "implementation": {
+                    "ot-telemetry": {
+                        "enabled":true,
+                        "package": "./telemetry/implementation/ot-telemetry.js",
+                        "config": {
```

Comment:
Use the same values for module configuration send Telemetry Data and signaling serverUrl are wrong for testnet and mainnet environment